ethexe/observer/src/lib.rs

```diff
@@ -81,7 +81,7 @@ pub enum ObserverEvent {
 // TODO (gsobol): make tests for observer service
 pub struct ObserverService {
     provider: Provider,
-    database: Box<dyn OnChainStorage>,
+    database: Database,
```

Comment:
mb just db: Database for code consistency

Reply:
done